core/src/test/java/org/elasticsearch/common/settings/ScopedSettingsTests.java

```diff
@@ -468,21 +468,21 @@ public void testDiff() throws IOException {
         ClusterSettings settings = new ClusterSettings(Settings.EMPTY, new HashSet<>(Arrays.asList(fooBar, fooBarBaz, foorBarQuux,
             someGroup, someAffix)));
         Settings diff = settings.diff(Settings.builder().put("foo.bar", 5).build(), Settings.EMPTY);
-        assertEquals(4, diff.size()); // 4 since foo.bar.quux has 3 values essentially
+        assertEquals(2, diff.size()); // 4 since foo.bar.quux has 3 values essentially
```

Comment:
these comments are no longer relevant i think? The essentially part was because each list element was it's own settings value?

Reply:
yeah correct I will remove it